src/radio/handlers.js

```diff
@@ -2,36 +2,34 @@
 
 import {Writable} from 'stream';
 import type {ConfigOptions} from '..';
-import soundcloud from './handlers/soundcloud';
-import youtube from './handlers/youtube';
+import SoundCloud from './handlers/soundcloud';
+import YouTube from './handlers/youtube';
 
-const handlers: Array<Class<Handler>> = [soundcloud, youtube];
+const handlers: * = [SoundCloud, YouTube];
```

Comment:
So is this just a typing thing to appeal to flow? I know * means sort of like, any type.

Reply:
Kinda sorta, yeah.

We had to litter things with `$FlowFixMe` because Flow's interfaces don't support typing on constructor and static method. I guessed that changing this to `*` is reasonably safe given that all handlers should be implementing `Handler` anyway, and Flow will complain loudly if anything doesn't implement what we actually use.